### PR TITLE
feat(New Features) : Revamp PA & FI API code snippets to support new features

### PR DESCRIPTION
**Pull request by vamshimupparaju**
As per new features in PA and FI end points added those new features to code snippets.

### CODE REVIEW

**Review by ksreeramoj — APPROVED** [review]

File: examples/src/examples/FiInteractiveEngineExample.java [hunk 1]
```diff
@@ -79,6 +90,8 @@ public static void main(String[] args) throws InterruptedException, JsonProcessi
       security2.setSettlement(FI_SETTLEMENT_2);
       security2.setDiscountCurve(FI_DISCOUNT_CURVE_2);
       security2.setSymbol(FI_SYMBOL_2);
+      security1.setBankLoans(fibankloans);
+      security1.setMunicipalBonds(fimunicipalbonds);
```
Comment (AkankshaAcharya) [inline examples/src/examples/FiInteractiveEngineExample.java:94]: these should be security2 right?

Comment (vamshimupparaju) [reply examples/src/examples/FiInteractiveEngineExample.java]: Updated

**Review by AkankshaAcharya — APPROVED** [review]

File: examples/src/examples/FiInteractiveEngineExample.java [hunk 2]
```diff
@@ -63,13 +63,24 @@ public static void main(String[] args) throws InterruptedException, JsonProcessi
...
       FICalculationParameters calcParameters = new FICalculationParameters();
       
+      FIBankLoans fibankloans = new FIBankLoans();
+      fibankloans.ignoreSinkingFund(true);
+      
+      FIMunicipalBonds fimunicipalbonds = new FIMunicipalBonds();
+      fimunicipalbonds.ignoreSinkingFund(true);
+      
+      FIMunicipalBondsForJobSettings fimunicipalbondsforjobsettings = new FIMunicipalBondsForJobSettings();
+      fimunicipalbondsforjobsettings.ignoreSinkingFund(true);
```
Comment (saigiridhar21) [inline examples/src/examples/FiInteractiveEngineExample.java:73]: Very minor: Can you please move these lines to 104 as they are specific to only job settings?

**Review by saigiridhar21 — APPROVED** [review]
Left some minor comments.

File: examples/src/examples/FiInteractiveEngineExample.java (hunk 2 above)
Comment (vamshimupparaju) [reply examples/src/examples/FiInteractiveEngineExample.java]: Updated

File: examples/src/examples/PAEngineMultipleUnitExample.java [hunk 3]
```diff
@@ -51,6 +51,9 @@ public class PAEngineMultipleUnitExample {
   private static String GROUP_NAME = "Economic Sector - FactSet";
   private static String GROUP_CATEGORY = "FactSet";
   private static String GROUP_DIRECTORY = "Factset";
+  private static String Pricing_Source_Name = "MSCI - Gross";
+  private static String Pricing_Source_Category = "MSCI";
+  private static String Pricing_Source_Directory = "Equity";
```
Comment (saigiridhar21) [inline examples/src/examples/PAEngineMultipleUnitExample.java:56]: Looks like the naming convention is incorrect.

File: examples/src/examples/PAEngineSingleUnitExample.java [hunk 4]
```diff
@@ -47,6 +47,9 @@ public class PAEngineSingleUnitExample {
   private static String GROUP_NAME = "Economic Sector - FactSet";
   private static String GROUP_CATEGORY = "FactSet";
   private static String GROUP_DIRECTORY = "Factset";
+  private static String Pricing_Source_Name = "MSCI - Gross";
+  private static String Pricing_Source_Category = "MSCI";
+  private static String Pricing_Source_Directory = "Equity";
```
Comment (saigiridhar21) [inline examples/src/examples/PAEngineSingleUnitExample.java:52]: Looks like the naming convention is incorrect.

Comment (vamshimupparaju) [reply examples/src/examples/PAEngineSingleUnitExample.java]: Updated naming conventions

File: examples/src/examples/PAEngineMultipleUnitExample.java (hunk 3 above)
Comment (vamshimupparaju) [reply examples/src/examples/PAEngineMultipleUnitExample.java]: Updated naming conventions

**Pull request merged**